### PR TITLE
Vy/add telemetry client

### PR DESCRIPTION
**Pull request by vix633**
Let's give this another whirl. Telemetry client could get called before it was initted :(

### CODE REVIEW

File: components/consistent-evaluation.js [hunk 1]
```diff
@@ -139,6 +146,9 @@ export class ConsistentEvaluation extends LitElement {
 			}
 		}
 		this._loading = false;
+		if (this._telemetry) {
+			this._telemetry.logLoadEvent('consistentEvalMain', this._submissionInfo.submissionList.length);
```
Comment (jmurray-d2l) [inline components/consistent-evaluation.js:150]: What happens if there are no submission?

Comment (vix633) [reply components/consistent-evaluation.js:150]: Can we even load this page without submissions?

Comment (jmurray-d2l) [reply components/consistent-evaluation.js:150]: Yep, we support no submissions - observational assignments are a thing we support

Comment (vix633) [reply components/consistent-evaluation.js:150]: Ah yeah we can, i'll add a null check!

**Review by jeroach — APPROVED** [review]
LGTM

**Review by jmurray-d2l — APPROVED** [review]

**Pull request merged**